lambdas/gro-pull-file/src/main/java/uk/gov/di/data/lep/GroPullFile.java

```diff
@@ -56,17 +56,19 @@ public GroFileLocations handleRequest(Object event, Context context) {
     }
 
     private RemoteFile downloadFile() {
+        var fingerprintId = config.getGroSftpServerFingerprintSecretID();
         var hostname = config.getGroSftpServerHost();
-        var privateKeyId = config.getGroSftpServerPrivateKeySecretId();
+        var privateKeyId = config.getGroSftpServerPrivateKeySecretID();
         var sourceDir = config.getGroSftpServerSourceDir();
         var username = config.getGroSftpServerUsername();
 
+        var fingerprint = awsService.getSecret(fingerprintId);
```

Comment:
I don't think this needs to be a secret - it should be no less public than the host details. No particular problem with doing it this way though

Reply:
What are your thoughts on making everything secret? So that we can configure without a redeploy